content/en/platform/corda/4.8/open-source/key-concepts-flows.md

```diff
@@ -20,78 +20,77 @@ title: Flows
 
-Rather than having to specify these steps manually, Corda automates the process using *flows*. A flow is a sequence
+Rather than having to specify these steps manually, Corda automates the process using **flows**. A flow is a sequence
 of steps that tells a node how to achieve a specific ledger update, such as issuing an asset or settling a trade.
 
-Here is the sequence of flow steps involved in the simple ledger update above:
+Here is a diagram showing the flow's steps used between Alice and Bob to perform the ledger update:
 
 {{< figure alt="flow sequence" width=80% zoom="/en/images/flow-sequence.png" >}}
 
 ## Running flows
 
-Once a given business process has been encapsulated in a flow and installed on the node as part of a CorDapp, the node’s
-owner can instruct the node to kick off this business process at any time using an RPC call. The flow abstracts all
-the networking, I/O and concurrency issues away from the node owner.
+Node operators use RPC calls to instruct their node to start a specific flow. The flow abstracts all
+the networking, I/O, and concurrency issues away from the node operator.
 
 All activity on the node occurs in the context of these flows. Unlike contracts, flows do not execute in a sandbox,
-meaning that nodes can perform actions such as networking, I/O and use sources of randomness within the execution of a
+meaning that nodes can perform actions such as networking, I/O, and use sources of randomness within the execution of a
 flow.
 
 ### Inter-node communication
 
-Nodes communicate by passing messages between flows. Each node has zero or more flow classes that are registered to
-respond to messages from a single other flow.
+Messages are passed from an active flow on one node to an active flow on another node. You can specify which flow classes
+a node can respond to and with what flow it responds with.
 
-Suppose Alice is a node on the network and wishes to agree a ledger update with Bob, another network node. To
-communicate with Bob, Alice must:
+For example, Alice is a node on the network and wishes to agree a ledger update with Bob, another network node. To
+communicate with Bob:
 
-* Start a flow that Bob is registered to respond to
-* Send Bob a message within the context of that flow
-* Bob will start its registered counterparty flow
+1. Alice starts a flow that Bob is registered to respond to.
+2. Alice sends Bob a message within the context of that flow.
+3. Bob starts its registered counterparty flow.
 
 Now that a connection is established, Alice and Bob can communicate to agree a ledger update by passing a series of
 messages back and forth, as prescribed by the flow steps.
 
 ### Subflows
 
 Flows can be composed by starting a flow as a subprocess in the context of another flow. The flow that is started as
-a subprocess is known as a *subflow*. The parent flow will wait until the subflow returns.
+a subprocess is known as a **subflow**. The parent flow will wait until the subflow returns.
```

Comment:
```suggestion
a subprocess is known as a *subflow*. The parent flow will wait until the subflow returns.
```
I think we introduce all new terms using italics.

Reply:
When you introduce a new concept that has a name you want users to remember, you can use bold to highlight this. In regular passages of prose, you don’t have to do it every time you mention that name, it’s just a way of drawing attention to something new.